paper-dropdown-menu.html

```diff
@@ -390,15 +399,17 @@
          * optional `label` property.
          */
         _selectedItemChanged: function(selectedItem) {
-          var value = '';
+          var value = '',
+              label = '';
           if (!selectedItem) {
-            value = '';
+            value = label = '';
           } else {
-            value = selectedItem.label || selectedItem.textContent.trim();
+            label = selectedItem.label || selectedItem.textContent.trim();
+            value = selectedItem[this.attrForSelected] || label;
```

Comment:
Would you consider the following: `value = selectedItem[this.contentElement.attrForSelected] || label;` ?

The reason being that `attr-for-selected` stays on the `paper-menu/paper-listbox` contained in the content section. This is useful since `paper-dropdown-menu` doesn't have any way of setting the selected item. Selection must be done on the `paper-listbox`, which means that one may have to duplicate the `attr-for-selected` on the `paper-listbox`.

It would enable the following: 
eg.)
```
let _bar = 0;
....
<paper-dropdown-menu>
  <paper-listbox selected="{{_bar}}" attr-for-selected="foo" class="dropdown-content">
    <paper-item foo="0">Zero</paper-item>
    <paper-item foo="10">Ten</paper-item>
    <paper-item foo="20">Twenty</paper-item>
  </paper-listbox>
</paper-dropdown-menu>
```